llvm/lib/Transforms/InstCombine/InstCombineShifts.cpp

```diff
@@ -1465,9 +1465,9 @@ Instruction *InstCombinerImpl::visitLShr(BinaryOperator &I) {
           MulC->logBase2() == ShAmtC) {
         // Look for a "splat" mul pattern - it replicates bits across each half
         // of a value, so a right shift is just a mask of the low bits:
```

Comment:
Comment needs to be updated. "mask of the low bits" describes the AND that was inserted before.